docs/pages/guides/CssOverrides.example.tsx

```diff
@@ -7,25 +8,25 @@ import { DatePicker, DatePickerProps } from '@material-ui/pickers';
 const materialTheme = createMuiTheme({
   overrides: {
     MuiPickersToolbar: {
-      toolbar: {
+      root: {
         backgroundColor: lightBlue.A200,
       },
     },
     MuiPickersCalendarHeader: {
-      switchHeader: {
+      root: {
         // backgroundColor: lightBlue.A200,
         // color: 'white',
       },
     },
     MuiPickersDay: {
-      day: {
+      root: {
         color: lightBlue.A700,
-      },
-      daySelected: {
-        backgroundColor: lightBlue['400'],
-      },
-      dayDisabled: {
-        color: lightBlue['100'],
+        '&$disabled': {
+          color: lightBlue['100'],
+        },
+        '&$selected': {
```

Comment:
Regarding the change of pseudo classes, I think that we could have handled in a different pull request, keeping our focus on a single topic at the time :)